### DISCUSSION TITLE
Release for IOS 16.2 live activity

### DISCUSSION THREAD

**Discussion by ahmedozy** [relayrides/pushy#1003] [Q&A] [answered]
When will be the the release containing the live activity, so it can be used through maven?

**Comment by jchambers** [comment, marked as answer]
Thank you for your enthusiasm. As I explained in https://github.com/jchambers/pushy/pull/990#issuecomment-1329968779, I will put out a new release as soon as I can. This has not changed. I understand that it's taking longer than originally estimated and that this is an important feature for many users and promise I will do this as soon as my schedule allows me to do so with quality.
